Directory.Build.props

```diff
@@ -12,7 +12,7 @@
     <EFVersion>6.1.3</EFVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' or '$(TargetFramework)' == 'netcoreapp3.0' ">
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' or '$(TargetFramework)' == 'netcoreapp3.0' or '$(TargetFramework)' == 'net6.0' ">
```

Comment:
I think EFVersion can be removed. So two here and where it's used, just put the version.

Reply:
I've made the changes as suggested.